android/src/main/cpp/MmkvHostObject.cpp

```diff
@@ -221,6 +221,20 @@ jsi::Value MmkvHostObject::get(jsi::Runtime& runtime, const jsi::PropNameID& pro
     });
   }
 
+  if (propName == "trim") {
+    // MMKV.trim()
```

Comment:
Just reference
* [MMKV/MMKV.h at master · Tencent/MMKV](https://github.com/Tencent/MMKV/blob/master/Core/MMKV.h#L310-L313)
* [MMKV/MMKV\_IO.cpp at master · Tencent/MMKV](https://github.com/Tencent/MMKV/blob/master/Core/MMKV_IO.cpp#L1008-L1046)

Reply:
wdym for reference?